src/features/offer/components/__tests__/OfferPartialDescription.web.test.tsx

```diff
@@ -9,55 +9,126 @@ import { OfferPartialDescription } from '../OfferPartialDescription'
     const { 'align-self': alignSelf } = offerSeeMoreContainer.style
     expect(alignSelf).toBe('center')
   })
-  it('places CTA on flex-end when provided a description', async () => {
-    const { getByTestId } = await renderOfferDescription(description)
+  it('places CTA on flex-end when provided a description', () => {
+    const { getByTestId } = renderOfferDescription(defaultParams)
     const offerSeeMoreContainer = getByTestId('offerSeeMoreContainer')
     // @ts-expect-error FIXME see how to fix typing
     const { 'align-self': alignSelf } = offerSeeMoreContainer.style
     expect(alignSelf).toBe('flex-end')
   })
-  it('renders externalLinks if http(s) url are present in the description', async () => {
-    const { queryByTestId } = await renderOfferDescription(description)
+  it('renders externalLinks if http(s) url are present in the description', () => {
+    const { queryByTestId } = renderOfferDescription(defaultParams)
     expect(queryByTestId('externalSiteIcon')).toBeTruthy()
   })
-  it("shouldn't render an empty line and a spacer when description is undefined", async () => {
-    const { queryByTestId } = await renderOfferDescription(undefined)
+  it("shouldn't render an empty line and a spacer when description is empty", () => {
+    const { queryByTestId } = renderOfferDescription({
+      ...defaultParams,
+      description: undefined,
+    })
     expect(queryByTestId('offerPartialDescriptionBody')).toBeFalsy()
   })
-  it("shouldn't render the SeeMore button if no content is on description page", async () => {
-    const { queryByTestId } = render(
-      // eslint-disable-next-line local-rules/no-react-query-provider-hoc
-      reactQueryProviderHOC(
-        <OfferPartialDescription id={offerId} description={undefined} />,
-        (queryClient: QueryClient) => {
+  describe('SeeMore button', () => {
+    it('should be rendered when there is some content on the description page', () => {
+      const { queryByTestId } = renderOfferDescription(defaultParams)
+      expect(queryByTestId('offerSeeMoreContainer')).toBeTruthy()
+    })
+    describe('should be rendered', () => {
+      it('when there is description on the description page (ellipsed or not : onTextLayout is not supported on the web)', () => {
+        const lines = [
+          { text: "Ce n'est pas le besoin d'argent où les " },
+          { text: 'vieillards peuvent appréhender de tomber ' },
+          { text: 'un jour qui les rend avares, car il y en a de ' },
+          { text: "tels qui ont de si grands fonds, qu'ils ne " },
+          { text: 'peuvent guère avoir cette inquiétude : et ' },
+          { text: "d'ailleurs comment pourraient-ils craindre " },
+          { text: 'de manquer dans leur caducité des ' },
+          {
+            text: "commodités de la vie, puisqu'ils s'en privent eux-mêmes volontairement pour satisfaire à leur avarice ?",
+          },
+        ]
+        const description = lines.map(({ text }) => text).join(' ')
```

Comment:
ici aussi, puis je n'ai pas vu si ca se répète mais si ca se répète dans plusieurs itération de test, tu peux créé une constante au début du test.